src/CoreWCF.Primitives/src/CoreWCF/Description/ServiceReflector.cs

```diff
@@ -229,7 +229,7 @@ internal static bool IsValidNCName(string name)
     }
 
     [DebuggerDisplay("{_encoded ?? _decoded}")]
-    internal class XmlName
+    public class XmlName
     {
```

Comment:
This shouldn't need to be public with the other changes I suggested.

Reply:
This was addressed in 75f6cfa72e482b0f0f42ad137be5f9b694044554.